### ISSUE TITLE
How to use Default Text Support with valid HTML?

### ISSUE THREAD

**Issue by brucek2** [harvesthq/chosen#2125]
When I start my single select list with `<option></option>`, the default text works as expected.

However, this HTML will no longer validate. The error is "Element option without attribute label must not be empty."

Is there a way to use the default text feature using only valid HTML syntax?

**Comment by sn3p** [comment]
+1

**Comment by DzmVasileusky** [comment]
Hi.
Adding label with one space helped for me 
`<option label=" " value=""></option>`

**Comment by tractorcow** [comment]
Yep I've run into this issue as well. Thanks to @DzmVasileusky for the solution!

**Comment by tractorcow** [comment]
Maybe https://github.com/harvesthq/chosen/blob/910a12599c10a11b7005feedb1e53e4b4535100e/coffee/chosen.jquery.coffee#L401 needs another condition? Perhaps it should check for empty option value rather than empty option abbr?

**Comment by tractorcow** [comment]
Just a guess, but does https://github.com/harvesthq/chosen/blob/dd269d4bef3d2118c00d51feebb370ab82967580/coffee/lib/abstract-chosen.coffee#L23 need to allow text = " " ?

**Comment by tractorcow** [comment]
I tested a fix locally, and changing the empty test from ` == ""` to `.match(/^\s*$/g)` (only whitespace) seems to work ok. I can use `&nbsp;` as a title for empty options, is html5 valid, and I get the expected behaviour from allow_single_deselect.

Now to learn coffeescript so I can PR this back....

**Comment by JoeSimsen** [comment]
Are there any plans to pull above fix/something similar into this project? Would really like to have it as well.
